packages/vue/docs/contributing/documentation-guidelines.md

```diff
@@ -1,3 +1,147 @@
-# Documentation guidelines
+# Documentation Guidelines
 
-TBD
\ No newline at end of file
+At Storefront UI, we know that documentation is key—both for users of the components as well as for (new and existing) contributors. We aim for complete, correct and easy to understand docs for our components and the project in general.
+
+This guide explains how different parts of the documentation play together and how contributors can help improving it.
+
+[[toc]]
+
+
+## The Basic Setup
+
+Different parts of documentation focus on different aspects, which is why they are located in various places:
+
+- The docs you read now are served via **VuePress**, a Markdown template-driven engine, perfect for general information about Storefront UI, mostly written in prose. The `.md` files (and some configuration files) located under _packages/vue/docs/_ serve this purpose.
+- **Storybook** stories may also be considered documentation, with a more formal and programmatic approach, focusing on characteristics of components. These `.stories.js` files reside next to the component's source code inside _packages/vue/src/components/_.
+- Some documentation may be found _inline_, i.e. **inside source code**. Important information for users is extracted automatically into other parts (e.g. slots are described in HTML comments but extracted into our main VuePress docs).
+
+
+## Prose Documentation
+
+Hand-written documentation (like the one you are reading now) takes place in Markdown files inside _packages/vue/docs/_. It follows less strict/formal rules but rather has to be addressed individually.
+
+
+## Storybook Stories
+
+::: warning WIP
```

Comment:
Do you want me to finish this?

Reply:
I must admit that I haven't looked too intense into the Storybook files yet. So if someone could take care of it (until I fully dig it) I'd really appreciate it 👏